Realm/RLMObjectBase.h

```diff
@@ -33,6 +33,9 @@ RLM_ASSUME_NONNULL_BEGIN
 
 + (NSString *)className;
 
+/// Returns whether the class is included in the default set of classes persisted in a Realm.
```

Comment:
If we're treating this as an official customization point then the docs need to describe it for a user overriding it, not for someone calling it. It also needs to be documented in RLMObject and Object, not here (RLMObjectBase is an implementation detail that should not appear anywhere in the documentation).